versions.tf

```diff
@@ -0,0 +1,3 @@
+terraform {
+  required_version = ">= 0.12.25"
```

Comment:
`required_version` is too high. The only new thing introduced in `0.12.25` of note was support for the `af-south-1` region, and this module doesn't depend on that.

Reply:
I picked a version that was on the flip side of a lot of tf bugs. I'm not sure testing for baseline min is worth it as tf had a _lot_ of regressions in between 0.12.9-0.12.19.